packages/suite/src/hooks/wallet/useSendForm.ts

```diff
@@ -283,8 +284,10 @@ export const useSendForm = (props: UseSendFormProps): SendContextValues => {
         }
     }, [getValues, composedLevels, signTransaction, resetContext, updateContext, goto]);
 
-    const typedRegister = useCallback(<T>(rules?: T) => register(rules), [register]);
-
+    const typedRegister = useCallback(
+        <T extends TypedValidationRules>(rules?: T) => register(rules as RegisterOptions),
```

Comment:
Can be changed to `RegisterOptions` everywhere, then there are no mismatches and it makes more sense IMO. Also there is no need in returing the `<Translation/>` component as an error, it can just be the key, then again, there are no type problems

Reply:
Not sure what do you mean, where else it can be changed? I don't want to change it anywhere else where it's not needed to make TS work after upgrade, some other refactors could be done in follow-up.

Reply:
Everywhere where `typedRegister` is used. I will make a commit :)

Reply:
Never mind, it was a bit different from what I assumed it was like. Still decided to simplify types b7160f658

Reply:
Cool thanks